CHANGELOG.md

```diff
@@ -5,78 +5,178 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 The crates in this repository do not adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) at this time.
 
-## Unreleased
-
-### Added
-- Support env overrides for ~all command-line flags.
-  - Flags that take multiple values can be repeated on the command line,
-    or passed as comma-separated values via environment or command-line args.
-
-### Changed
-
-### Rust Dependencies
-* Updated `rust-toolchain` version to newer nightly
-  * enables use of [Generic Associated Types](https://github.com/rust-lang/rust/issues/44265) and [static async fn in traits](https://github.com/rust-lang/rust/issues/91611)
-* Replaced `datatest` with a custom `test_with_data` macro.
-* Replace `structopt` with `clap`.
-* Updated grpcio from 0.9 to 0.10.
-
 ## [1.2.0]
 
 ### Added
 
- - Encrypted Memos ([MCIP #3](https://github.com/mobilecoinfoundation/mcips/pull/3))
- - Recoverable Transaction History ([MCIP #4](https://github.com/mobilecoinfoundation/mcips/pull/4))
- - Consensus/Fog repository merge
+- Fog is now part of this repository
+- Fog ledger key image checks are now oblivious ([fog#101])
+- Fog View OMAP size configurable via environment
+- Fog Overseer utility for monitoring Fog Ingest instances
+- Fog Ingest Client CLI now allows queries to the `get_ingress_key_record` API
+- Block versioning / protocol evolution ([mcips#26])
+- In Block Version 1 (to be enabled along with block version 2):
+  - Required Transaction Memos ([mcips#3])
+  - Recoverable Transaction History ([mcips#4])
+- In Block Version 2 (to be enabled after network has been upgraded):
+  - Confidential Multi-Token Support ([mcips#25])
+  - Minting support for non-MOB tokens
+  - Verifiable burning for any token ([mcips#35])
+  - Standardized Addresses for common purposes ([mcips#36])
+- In Block Version 3 (will be enabled in a future release):
+  - Require TxOuts to be sorted in TxProposal.
 
 ### Changed
 
- - Updated SGX to 2.15
- - Lock enclave no-debug mode when building for IAS production.
- - Update Rust toolchain to `nightly-2021-07-21`.
-
-#### Rust Dependencies
+- Use mnemonics instead of the deprecated root entropy derivation when generating test keys/ledgers. ([#1893])
+- Enable `Bitcode` for `libmobilecoin`, reduce mobile artifact size by ~25% ([#1124])
+- Make fog ingest state file handling more resilient ([#1358])
+- Fee-accumulation in the enclave is now constant-time ([#1780])
+- mobilecoind will now exit on startup when a ledger migration is necessary, unless the new `--ledger-db-migrate` command line argument is used, in which case it will migrate automatically
```

Comment:
```suggestion
- mobilecoind will now exit on startup when a ledger migration is necessary, unless the new `--ledger-db-migrate` command line argument is used, in which case it will migrate automatically. This flag does not do anything if the Ledger DB does not exist.
```